### PR TITLE
chore: sign windows installer

### PR DESCRIPTION
**Pull request by malandis**
We use a code signing certificate to sign the windows binary and windows installer. We also refactor the publish-windows-assets job steps to be more self contained.

### CODE REVIEW

File: .github/workflows/execute-release.yml [hunk 1]
```diff
@@ -248,65 +248,123 @@ jobs:
...
+      - name: Delete PFX certificate
+        env:
+          CERT_PATH: ${{ steps.write_pfx.outputs.pfx_path }}
+        run: Remove-Item -Path $env:CERT_PATH
+
+      - name: Create zip and installer urls
+        id: create_release_urls
+        env:
+          ZIP_FILENAME: ${{ steps.create_zip.outputs.asset_name_zip }}
+          MSI_FILENAME: ${{ steps.create_zip.outputs.asset_name_msi }}
```
Comment (pgautier404) [inline .github/workflows/execute-release.yml:340]:
```suggestion
          MSI_FILENAME: ${{ steps.build_installer.outputs.asset_name_msi }}
```

Comment (malandis) [reply .github/workflows/execute-release.yml]: Thank you! Fixed in [36080c6](https://github.com/momentohq/momento-cli/pull/219/commits/36080c6875d3ea674748dc9a8e9786f87270c780)

**Review by pgautier404 — APPROVED** [review]
This is a tremendous improvement, thanks!

**Review by pgautier404 — DISMISSED** [review]

**Review by cprice404 — COMMENTED** [review]
This looks great to me but before we merge it, we need to think about what will happen if the signing fails (I believe we haven't been able to test it on github yet, right?)

If it fails after the github release is created, then the CLI canary will fail, which will cause problems in production.

So we need to be sure that either (a) if it fails, it will fail before the release is created, or (b) we have tested the signing code prior to merging this.

If you need some ideas on how to test the signing code without going through the full release process let me know, I can help.

**Comment by malandis** [comment]
> This looks great to me but before we merge it, we need to think about what will happen if the signing fails (I believe we haven't been able to test it on github yet, right?)
> 
> If it fails after the github release is created, then the CLI canary will fail, which will cause problems in production.
> 
> So we need to be sure that either (a) if it fails, it will fail before the release is created, or (b) we have tested the signing code prior to merging this.
> 
> If you need some ideas on how to test the signing code without going through the full release process let me know, I can help.

@cprice404 the build passes here https://github.com/momentohq/momento-cli/actions/runs/3843526807/jobs/6545864685

**Review by pgautier404 — APPROVED** [review]

**Pull request merged**